File: Softwares/DailyDB/App_Code/Model/Verificacion.cs
using System;

namespace DailyDB.App_Code.Model
{
    /// <summary>
    /// Summary description for Verificacion
    /// </summary>
    public class Verificacion
    {
        public Verificacion()
        {
        }

        public int VerificacionId { get; set; }
        public String CodigoVerificacion { get; set; }
        public DateTime HorarioInicio { get; set; }
        public DateTime HorarioFin { get; set; }
        public int UsuarioId { get; set; }
        public Boolean Estado { get; set; }

    }
}

File: Android Applicacion/Pruebas/app/src/main/java/daily/pruebaconexion/Modelo/Verificacion.java
package daily.pruebaconexion.Modelo;

import java.util.Date;

public class Verificacion {
    public int VerificacionId;
    public String CodigoVerificacion;
    public Date HorarioInicio;
    public Date HorarioFin;
    public int UsuarioId;
    public boolean Estado;


    public int getVerificacionId() {
        return VerificacionId;
    }

    public void setVerificacionId(int verificacionId) {
        VerificacionId = verificacionId;
    }

    public String getCodigoVerificacion() {
        return CodigoVerificacion;
    }

    public void setCodigoVerificacion(String codigoVerificacion) {
        CodigoVerificacion = codigoVerificacion;
    }

    public Date getHorarioInicio() {
        return HorarioInicio;
    }

    public void setHorarioInicio(Date horarioInicio) {
        HorarioInicio = horarioInicio;
    }

    public Date getHorarioFin() {
        return HorarioFin;
    }

    public void setHorarioFin(Date horarioFin) {
        HorarioFin = horarioFin;
    }

    public int getUsuarioId() {
        return UsuarioId;
    }

    public void setUsuarioId(int usuarioId) {
        UsuarioId = usuarioId;
    }

    public boolean isEstado() {
        return Estado;
    }

    public void setEstado(boolean estado) {
        Estado = estado;
    }
}
